### PR TITLE
Localize tips

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR
Localize all tips

## Why / Balance
I don't see why it shouldn't be localized.

### CODE REVIEW

**Comment by mirrorcult** [comment]
really this is just a larger issue where string datasets are not easily localizable, ideally the dataset proto would stay the same and the system had a way to override it with loc ids or something

File: Resources/Prototypes/Datasets/tips.yml [hunk 1]
```diff
@@ -1,137 +1,138 @@
...
-  - "There is a chemical metabolism limit that limits the amount of reagents of a certain type you can digest at once. Certain species have higher metabolism limits, such as slimes."
-  - "Welding without proper eye protection can cause eye damage, which must be cured with oculine."
-  - "Zombies are very vulnerable to heat damage, making welding tools and laser guns extremely effective against them."
-  - "You can weld glass shards into glass sheets."
-  - "By right clicking on a player, and then clicking the heart icon, you can quickly examine them to check for injuries or how badly they're bleeding. You can also do this to yourself."
-  - "Monkeys and kobolds have a rare chance to be sentient. Ook!"
-  - "You can tell if an area with firelocks up is spaced by looking to see if the firelocks have lights beside them."
-  - "Instead of picking it up, you can alt-click food to eat it. This also works for mice and other creatures without hands."
+  - tips-dataset-1
+  - tips-dataset-2
```
Comment (deltanedas) [inline Resources/Prototypes/Datasets/tips.yml:5]: dont do this copy paste N times shit just have it be `type: dataset count: 135` for datasets that are required to be localized, at the very least

Comment (mirrorcult) [reply Resources/Prototypes/Datasets/tips.yml:5]: yeah i would probably do something like this

Comment (Tayrtahn) [reply Resources/Prototypes/Datasets/tips.yml:5]: > dont do this copy paste N times shit just have it be `type: dataset count: 135` for datasets that are required to be localized, at the very least

Working on something very close to this.

**Comment by Tayrtahn** [comment]
I gotchu fam: #28310

**Review by AJCM-git — DISMISSED** [review]
merge master and use #28310

**Review by Tayrtahn — APPROVED** [review]

**Comment by amo020** [comment]
Just used it on an ongoing fork. Works like a charm

**Pull request merged**